site/src/ThemeSelector.jsx

```diff
@@ -0,0 +1,46 @@
+import PropTypes from "prop-types";
+import { useState, useEffect } from "react";
+import LayerIcon from "./icons/icon-layers.svg?react";
```

Comment:
@eti7075 - you will need to make a similar import statement for the 'rotate' icon, which converts the icon to a full-fledge react component. 

The reason this is good- is that if you do 

`<img src="someIcon.svg"/>` you cannot 'reach inside' the SVG to style individual lines therein. 

By converting the SVG to a react component, the raw <svg> tag is embedded in the DOM, allowing you to target both the 'primary' and 'secondary' paths in it. 

Try viewing the source of the refresh icon I sent you and you'll see one `circle` that's classed 'primary', and one `path` that's classed 'secondary'.